examples/coexist_klog_v1_using_logr/coexist_klog_v1.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"flag"
+	"github.com/go-logr/logr"
+
+	"k8s.io/klog"
+	klogv2 "k8s.io/klog/v2"
+)
+
+type klogLogger struct{}
+
+func (kw klogLogger) Info(msg string, keysAndValues ...interface{}) {
+	klog.InfoDepth(5, msg)
+}
+
+func (kw klogLogger) Enabled() bool {
+	return true
+}
+
+func (kw klogLogger) Error(err error, msg string, keysAndValues ...interface{}) {
+	klog.ErrorDepth(5, msg)
+}
+
+func (kw klogLogger) V(level int) logr.InfoLogger {
+	return kw
+}
+
+func (kw klogLogger) WithValues(keysAndValues ...interface{}) logr.Logger {
+	return kw
+}
+
+func (kw klogLogger) WithName(name string) logr.Logger {
+	return kw
+}
```

Comment:
I'm confused about why discarding the data is ok... isn't the goal of this example is to show how klog/v2 and klog/v1 can coexist and work properly? This seems like it routes log data to the same destination at the price of dropping the data or changing verbosity behavior...

Reply:
Yep same as above. the SetOutput one came out better than this one.